supervisely/api/video/video_annotation_api.py

```diff
@@ -254,18 +252,28 @@ async def download_async(
         video_id: int,
         video_info=None,
         semaphore: Optional[asyncio.Semaphore] = None,
+        force_metadata_for_links: bool = True,
+        integer_coords: bool = True,
```

Comment:
typing: Optional

Reply:
Optional[X] is equivalent to Union[X, None]
That's not the case.